docker-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+# until psql $DATABASE_URL -c '\l'; do
+#   >&2 echo "Postgres is unavailable - sleeping"
+#   sleep 1
+# done
+
+# >&2 echo "Postgres is up - continuing"
+
+if [ "x$DJANGO_MANAGEPY_MIGRATE" = 'xon' ]; then
+    python /imagetagger/imagetagger/manage.py migrate --noinput
```

Comment:
This is not expensive. Why don't you run this command on *every* run of the container? This can prevent a lot of problems due to missing migrations on updates. In addition, it would simplify things as $DJANGO_MANAGEPY_MIGRATE would not be required anymore.